core-web/libs/portlets/edit-ema/portlet/src/lib/shared/models.ts

```diff
@@ -1,4 +1,4 @@
-import { UVE_MODE, CLIENT_ACTIONS } from '@dotcms/client';
+import { CLIENT_ACTIONS } from '@dotcms/client';
```

Comment:
This will be `@dotcms/client/types` later no?

Reply:
Yes sir